Python and Jupyter updates (#1478)
duckdb/duckdb-web: docs/guides/python/jupyter.md

LIMIT 5
```

> To return the query results into a Python variable, use `<<` as an assignment operator.

Reviewer: ```suggestion
To store the query results in a Python variable, use `<<` as an assignment operator.
```
